rpmconf/Install/zmsetup.pl

```diff
@@ -7102,6 +7108,10 @@ sub applyConfig {
     # 32295
     setLdapGlobalConfig("zimbraSkinLogoURL", "http://www.zimbra.com")
       if isFoss();
+
+    if ($newinstall){
```

Comment:
can we remove the check for newinstall and add check for if ldap installed?

Reply:
Isn't that a bad idea since you wouldn't be able to know if a transition from one schema version to another was occurring?

Is this capability not desired now or in the future?
Blindly inserting this value during setup if ldap is installed would prevent this from happening since there would be no knowledge of the previous value.

Reply:
FYI: This check is already inside the 'if (isEnabled("zimbra-ldap")' guarded section.

If anything the '$newinstall' check should be removed if that is desired.